app/views/shared/_error_messages.html.erb

```diff
@@ -0,0 +1,12 @@
+<% if @user.errors.any? %>
+  <div id="error_explanation">
+    <div class="alert alert-danger">
+      <%= t("shared._error_messages.text1") %> <%= pluralize(@user.errors.count, "error") %>
```

Comment:
1. `"error"` I18n nè em

2. Phân biệt cho chị size, length và count
Cân nhắc là dùng count ở đây hợp lý chưa nhé

Reply:
Trước hết xét trong array:
- Về cơ bản, length trong array sẽ trả về số phần tử trong mảng.
- array#size chỉ là alias của array#length method. Nó có cùng tính năng và cách thực hiện như array#length.
- Khác với length và size. array#count có thể sử dụng đi kèm điều kiện để tính toán số phần tử. Có ba cách gọi như sau:
    Array#count trả về số phần tử trong mảng
    Array#count n trả về số phần tử có giá trị = n trong mảng
    Array#count{|i| i.even?} trả về số phần tử trong mảng dựa trên điều kiện.
Nhưng nếu xét về việc sử dụng các Method trên trong ActiveRecord thì size sẽ là lựa chọn tối ưu hơn vì nó có cách thức hoạt động trả về giá trị nhanh và thông minh hơn length và count.